x-pack/plugins/siem/server/lib/ip_details/elasticsearch_adapter.ts

```diff
@@ -32,8 +39,12 @@ import {
   IpDetailsAdapter,
   IpOverviewHit,
   OverviewHit,
-  UsersBucketsItem,
+  TlsBuckets,
 } from './types';
+
+import { buildTlsQuery } from './query_tls.dsl';
+import { UsersBucketsItem } from './types';
```

Comment:
These imports can be optimized -- there are duplicate imports for `'./types'` & `'./query_tls.dsl'`. Looks like an `es-lint --fix` doesn't take care of them either... 😔 Was able to fix them with an `optimize imports` via my IDE though.

Reply:
Yikes! I see what you mean, I fixed at least one I think here. Let me know if you see others. I am hopeful we can get back some of these linter rules soon.